docs/topics/extraction.md

```diff
@@ -92,6 +126,18 @@ git remote rm repo-source
 git tag | xargs git tag -d
 ```
 
+### 6. Create a virtualenv and activate it
+
+To create the virtualenv, it is recommended to use the same Python version (MAJOR.MINOR) as the one [listed here](https://github.com/saltstack/salt/blob/master/cicd/shared-gh-workflows-context.yml).
+
+```shell
+mkdir workdir && cd workdir
+python3.10 -m venv venv --prompt saltext-stalekey
+source venv/bin/activate
+```
+
+Please ensure you're inside your virtual environment from here on.
+
```

Comment:
Any reasons why you moved it here? Maybe then it is necessary to only recommend `pipx` to  install the tools and not `pip`?

Reply:
The reason was that - following the previous steps verbatim - the venv ended up outside of the project root (in `workdir/venv`, not `workdir/saltext-stalekey/venv`), which I thought might be confusing, especially since other guides always put it in the latter location.

Yeah, maybe we should at least add `pipx` as the default for `git-filter-repo` installation, similar to the above step.

Reply:
I removed `mkdir workdir && cd workdir`, which was incorrectly left in place.

Also added the preferred `pipx` installation for `git-filter-repo` to hint at installing to a non-system location.